File: app/src/main/java/com/example/jaguar/weather/model/network/WeatherService.java
package com.example.jaguar.weather.model.network;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

    public interface WeatherService {
        @GET("forecast.json")
        Call<UserModelResponse> getWeather(@Query("key") String key,
                                           @Query("q") String q,
                                           @Query("lang") String lang,
                                           @Query("days") String days);
    }

File: app/src/main/java/com/example/jaguar/weather/model/network/WeatherService.kt
package com.example.jaguar.weather.model.network

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface WeatherService {
    @GET("forecast.json")
     fun getWeather(@Query("key") key: String,
                            @Query("q") q: String,
                            @Query("lang") lang: String,
                            @Query("days") days: String): Call<UserModelResponse>
}